Sources/StreamChatUI/ChatMessageList/Attachments/FileAttachmentViewInjector.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// The delegate used `GiphyAttachmentViewInjector` to communicate user interactions.
+public protocol FileActionContentViewDelegate: ChatMessageContentViewDelegate {
+    /// Called when the user taps on attachment action
+    func handleFileAttachmentTap(_ attachment: ChatMessageFileAttachment)
```

Comment:
let's follow the `didTap` pattern, in this case, it would be:
```
func didTapOnAttachment(_ attachment: ChatMessageFileAttachment, at indexPath: IndexPath)
```

Reply:
@DominikBucher12 the attachment doesn't have a backreference to its message. How would you know to which message the attachment belongs?

Reply:
Okay, I edited the function, thanks :)